setup.py

```diff
@@ -14,10 +14,14 @@
         'pandas',
         'click',
         'pyyaml',
+        'pillow',
```

Comment:
Note that you also need to add these to `environment.yml` so they get installed in the automated tests.